.github/workflows/vitest-changed.yml

```
@@ -221,291 +225,291 @@ jobs:
+  #       working-directory: packages/agent-builder/integration-tests
+
+  #     - name: Run agent-builder integration tests
+  #       if: steps.check-changes.outputs.changed == 'true'
+  #       run: pnpm test
+  #       working-directory: packages/agent-builder/integration-tests
+  #       env:
+  #         NODE_OPTIONS: '--max_old_space_size=6144'
+  #         OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
+
+  # observability-test:
     name: Observability Tests
```

actionlint (1.7.10)

[error] key "name" is duplicated in "e2e-test" job. previously defined at line:122,col:5

(syntax-check)